catt/controllers.py

```diff
@@ -26,6 +26,20 @@ def get_stream_info(video_url):
     return stream_info
 
 
+def get_chromecast(device_name):
+    devices = pychromecast.get_chromecasts()
+    if device_name:
```

Comment:
Sure thing, but I think we should at least print the device name when it has been specified, so people know that it's in the configuration. So, if you've specified a device, we should print at the start "Trying to connect to <name>..." or something like that.